### PR TITLE
added example that uses 2 authorisers

### PR DESCRIPTION
**Pull request by bjartek**
I did not make an issue for this should I do that?

## Description

There are no examples on how to get two authorisers in a transaction. This fixes that

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-go-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Comment by cybercent** [comment]
Instead of updating an existing use case, you could just add a new one with 2 authorizers.

**Comment by bjartek** [comment]
Will do!

**Comment by bjartek** [comment]
I have not created a new example and not modified the old one at all.

File: README.md [hunk 1]
```diff
@@ -335,6 +336,57 @@ err = tx.SignEnvelope(account2.Address, key3.ID, key3Signer)
...
+key3 := account2.Keys[0]
+
+// create signers from securely-stored private keys
+key1Signer := getSignerForKey1()
+key3Signer := getSignerForKey3()
+
+tx := flow.NewTransaction().
+    SetScript([]byte(`
+        transaction {
+            prepare(signer: AuthAccount, signer2:AuthAccount) {
```
Comment (psiemens) [inline README.md:368]:
```suggestion
            prepare(signer1: AuthAccount, signer2: AuthAccount) {
```

File: examples/transaction_signing/multi_party_two_authorizers/main.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+	key3Signer := crypto.NewInMemorySigner(privateKey3, key3.HashAlgo)
+
+	account1 := examples.CreateAccount(flowClient, []*flow.AccountKey{key1}, nil)
+	account2 := examples.CreateAccount(flowClient, []*flow.AccountKey{key3}, nil)
+
+	tx := flow.NewTransaction().
+		SetScript([]byte(`
+		transaction { 
+			prepare(signer: AuthAccount, signer2:AuthAccount) {
```
Comment (psiemens) [inline examples/transaction_signing/multi_party_two_authorizers/main.go:68]:
```suggestion
			prepare(signer1: AuthAccount, signer2: AuthAccount) { 
```

**Review by psiemens — APPROVED** [review]
LGTM!

**Pull request merged**